Tasks/PublishCodeCoverageResults/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -1,14 +1,16 @@
 {
   "loc.friendlyName": "Publish Code Coverage Results",
   "loc.helpMarkDown": "[More Information](https://go.microsoft.com/fwlink/?LinkID=626485)",
-  "loc.description": "Publish code coverage results to VSTS/TFS",
-  "loc.instanceNameFormat": "Publish Code Coverage Results $(summaryFileLocation)",
+  "loc.description": "Publish Cobertura or JaCoCo code coverage results from a build",
+  "loc.instanceNameFormat": "Publish code coverage from $(summaryFileLocation)",
   "loc.input.label.codeCoverageTool": "Code Coverage Tool",
-  "loc.input.help.codeCoverageTool": "Select the tool with which code coverage summary files have been generated",
+  "loc.input.help.codeCoverageTool": "Select the tool with which code coverage results are generated.",
   "loc.input.label.summaryFileLocation": "Summary File",
-  "loc.input.help.summaryFileLocation": "Path of code coverage summary file, which has code coverage statistics like line, method, class coverage. For example, $(Build.SourcesDirectory)/CodeCoverage/Summary.xml",
+  "loc.input.help.summaryFileLocation": "Enter the path of the summary file containing code coverage statistics, such as line, method, and class coverage. The value may contain minimatch patterns. For example, `$(System.DefaultWorkingDirectory)/MyApp/**/site/cobertura/coverage.xml`",
```

Comment:
In most other Tasks we use the passive / descriptive form over the active form:
"Path of the summary file..."
vs.
"Enter the path..."

How important is consistency?